### DISCUSSION TITLE
ibis-trino backend - experimental warning

### DISCUSSION THREAD

**Discussion by vmsaipreeth** [ibis-project/ibis#8690] [Q&A] [answered]
Hi Team,

For [ibis-trino](https://ibis-project.org/backends/trino), currently we see the below warning.

```
Warning
This backend is experimental and is subject to backwards incompatible changes
```

When can we expect this warning to be removed? Or can we consider trino backend as stable?

**Comment by jcrist** [comment, marked as answer]
Thanks for raising this - the trino backend (and really all our released backends) should be considered stable - I've pushed up a PR [here](https://github.com/ibis-project/ibis/pull/8691) to remove those warnings.

**Reply by vmsaipreeth** [reply]
Thanks for a quick response, @jcrist !
